src/services/user/setFollowerCount.js

```diff
@@ -0,0 +1,21 @@
+const {successResponse, errorResponse} = require('../../utils');
+const UsersFunction = require('../../databases/functions/users');
+
+const setAllFollowerCount = async (req, res) => {
+  try {
+    try {
+      await UsersFunction.setAllFollowerCount();
+    } catch (error) {
+      return errorResponse(res, 'Failed to set follower count', 500);
+    }
+
+    return successResponse(res, 'set follower count was successful');
+  } catch (error) {
+    console.log(error);
+    return errorResponse(res, error.toString(), 500);
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is a redundant nested try-catch block in your code. You can simplify it by using only one try-catch block. Also, it's not a good practice to log the error directly to the console. Instead, consider using a logging library that can provide more context and control over what gets logged and where it goes.

```diff
-  try {
-    try {
-      await UsersFunction.setAllFollowerCount();
-    } catch (error) {
-      return errorResponse(res, 'Failed to set follower count', 500);
-    }
-
-    return successResponse(res, 'set follower count was successful');
-  } catch (error) {
-    console.log(error);
-    return errorResponse(res, error.toString(), 500);
-  }

+  try {
+    await UsersFunction.setAllFollowerCount();
+    return successResponse(res, 'set follower count was successful');
+  } catch (error) {
+    // Use a logger here instead of console.log
+    // logger.error(error);
+    return errorResponse(res, 'Failed to set follower count', 500);
+  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->